docs/eventing/brokers/broker-types/mt-channel-based-broker/README.md

```diff
@@ -0,0 +1,117 @@
+# Multi-tenant channel based Broker
+
+The Multi-tenant channel based Broker (MTChannelBasedBroker) uses [Channels](../../../channels) for event routing. It is shipped by default with Knative Eventing.
+Users should prefer native Broker implementations over the MTChannelBasedBroker and channel combo because it is usually more efficient. 
+## Prerequisites
+
+You need to have Knative Eventing installed.
+
+## Install a channel implementation
+
+As the MTChannelBasedBroker is based on Channels, you need to install a Channel implementation. Check out the [available Channels](../../../channels/channels-crds.md) for a (non-exhaustive) list of the available Channels for Knative Eventing.
+
+You can install e.g. the in-memory channel via:
+
+```bash
+kubectl apply -f {{ artifact(repo="eventing",file="in-memory-channel.yaml")}}
+```
+
+## Create a MTChannelBasedBroker
+
+You can create a MTChannelBasedBroker by using the `kn` CLI or by applying YAML files using `kubectl`.
+
+=== "kn"
+
+    You can create a MTChannelBasedBroker by entering the following command:
+
+    ```bash
+    kn broker create <broker-name> --class MTChannelBasedBroker
+    ```
+
+=== "kubectl"
+
+    The YAML in the following example creates a broker named `default`.
+
+    1. Create a MTChannelBasedBroker by creating a YAML file using the following template:
+
+        ```yaml
+        apiVersion: eventing.knative.dev/v1
+        kind: Broker
+        metadata:
+          annotations:
+            eventing.knative.dev/broker.class: MTChannelBasedBroker
+          name: <broker-name>
+        ```
+  
+        !!! note
+            Note, that the broker class is specified via the `eventing.knative.dev/broker.class` annotation (as for all Broker types).
```

Comment:
Please don't use parenthesis. See https://github.com/knative/docs/blob/main/contribute-to-docs/style-guide/style-and-formatting.md#only-use-parentheses-for-acronym-explanations

Reply:
:+1: Done